ReactWindows/ReactNative.Shared/Views/Text/ReactHyperlinkViewManager.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.UIManager.Events;
+using Windows.UI.Xaml;
+#if WINDOWS_UWP
```

Comment:
Does this work for WPF? If not why bother to put it in shared?

Reply:
Someone like @piyush or @pre10der89 might want to submit a WPF version to your branch, or soon after.

Reply:
It should. I need to move one of the usings.